tests/e2e/scripts/cloud-run/deploy.sh

```diff
@@ -324,7 +324,7 @@ function setup() {
     -i "${APIPROXY_IMAGE}"
 
   # Redeploy ESPv2 to update the service config
-  proxy_args="^++^--tracing_sample_rate=0.01++--tracing_outgoing_context=x-cloud-trace-context"
+  proxy_args="^++^--tracing_sample_rate=0.05++--tracing_outgoing_context=traceparent++--tracing_incoming_context=traceparent"
```

Comment:
Why don't we set tracing_incoming_context before?

Reply:
Technically it's not needed yet. But soon Cloud Run will start publishing traces as well, and we want to keep propagating the trace context.

Reply:
This will matter as soon as b/151160669 is fixed. Seems like there is progress maybe made on it.